tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = 
+#     py36-django{11,20},
+#     flake8,
+    apidocs
+
+skip_missing_interpreters = True
+usedevelop = True
+
+# [testenv]
+# deps =
+#     django11: Django<2.0
+#     django20: Django>=2.0,<2.1
+# 
+# envlist = 
+#     py36-django{11,20}
+# 
+# commands =
+#     python setup.py test
+
+[testenv:flake8]
+deps = flake8
+commands = flake8 --ignore=W391 djongo {posargs}
+
+[testenv:apidocs]
+whitelist_externals =
+    rm
+    cd
+    mkdir
+commands =
+	rm -f apidocs/modules.rst
+	mkdir -p apidocs/_static
+	sphinx-apidoc -o apidocs/ djongo
+	rm -f apidocs/modules.rst
```

Comment:
We can either run the commands from tox to build the docs, or have a separate python script to do this. 

I just called the script from tox, as I was using the script at times to build, just the docs, from the commandline. I have made something like that in the main branch, what is your opinion?

Reply:
Combining a bit might be good but it always feels wrong to me to execute shell scripts from Python, but that's just a personal preference ;)